AK/FuzzyMatch.h

```diff
@@ -19,126 +18,12 @@ struct FuzzyMatchResult {
 static constexpr int const RECURSION_LIMIT = 10;
 static constexpr int const MAX_MATCHES = 256;
```

Comment:
Done!

Reply:
In case you're curious about the backstory: While making that edit, I initially left in the "short" version of the function, i.e. `fuzzy_match(String const& needle, String const& haystack)`. This function is just a thin wrapper around the full interface, using `MAX_MATCHES` in its definition. For some weird reason I also assumed that `RECURSION_LIMIT` would be the default recursion limit and somehow used in that function definition. At that point in time, it was a reasonably good idea to keep it in.

Then I decided to just move it out of line, because fuzzy_match has no business being so performance critical to slow down compilation that much, and I didn't notice that the constexpr-s became totally superfluous.

In case you're _not_ curious about the backstory, here are some happy cows: https://www.redditp.com/r/happycowgifs